checkov/kubernetes/parser/parser.py

```diff
@@ -21,9 +22,10 @@ def parse(filename: str) -> tuple[list[dict[str, Any]], list[tuple[int, str]]] |
         if filename.endswith(".json"):
             (template, template_lines) = k8_json.load(Path(filename))
         if template:
+            k8s_validator = K8sValidator()
             if isinstance(template, list):
                 for t in template:
-                    if t and isinstance(t, dict) and 'apiVersion' in t.keys() and 'kind' in t.keys():
+                    if k8s_validator.is_valid_template(t):
```

Comment:
```suggestion
                    if K8sValidator.is_valid_template(t):
```
if you are using a static method you don't need to create an instance of the class for it